c2corg_ui/static/partials/cards/articles.html

```diff
@@ -0,0 +1,20 @@
+<a ng-href="{{cardCtrl.createURL()}}" ng-init="doc = cardCtrl.doc; locale = cardCtrl.locale;">
+  <div class="article-author text-center">
```

Comment:
in this case, maybe not the most correct and consistent way of doing, but you would have to keep the classes with `outing-` because they are defined and styled in the CSS whereas `article-` are not and will not display correctly.